pinakes/main/approval/services/update_request.py

```diff
@@ -182,10 +183,13 @@ def _update_parent(self):
 
     # start the external approval process if configured
     def _run_request(self):
-        validations.runtime_validate_group(self.request)
+        if not validations.runtime_validate_group(self.request):
+            return
```

Comment:
can we add log messages here when validation fails?

Reply:
Added logging in the validation method.